### PR TITLE
New functions for using API to connect Openshift+CNV

### PR DESCRIPTION
**Pull request by eko999**
With the new functions, it's easy get the attributes for guest and its worker node.
A demo as:
```
from virt_who import *
from virt_who.base import Base
from virt_who.register import Register
from virt_who.provision import Provision

class RunProvision(Provision):
    def test_run(self):
        guest_name = 'rhel83-vm'
        version = self.kubevirt_version()
        guest_attrs = self.kubevirt_guest_attrs(guest_name)
        host_attrs = self.kubevirt_host_attrs(guest_attrs['guest_node'])
        guest_ip = self.kubevirt_guest_ip(guest_name)
        logger.info(version)
        logger.info(guest_attrs)
        logger.info(host_attrs)
        logger.info(guest_ip)

if __name__ == "__main__":
    unittest.main()
```

The output as:
```
# nosetests-3 demo.py
2021-01-21 22:59:59 [INFO] v1alpha3
2021-01-21 22:59:59 [INFO] {'guest_id': '1a559ee9-9aa4-5d2d-bc6c-ecdb8d8c8d28', 'guest_node': 'worker-1.virtwho1.lab.upshift.rdu2.redhat.com', 'guest_status': 'Running'}
2021-01-21 22:59:59 [INFO] {'host_uuid': '09cf0137bde545eb885bcadda61c84e2', 'host_name': 'worker-1.virtwho1.lab.upshift.rdu2.redhat.com'}
2021-01-21 22:59:59 [INFO] worker-1.virtwho1.lab.upshift.rdu2.redhat.com:32516
```

### CODE REVIEW

**Comment by hkx303** [comment]
That's great progress for the new way to connect CNV env.

File: utils/installer.py [hunk 1]
```diff
@@ -258,17 +258,17 @@ def setup_libvirt():
 
 
 def setup_kubevirt():
-    master = deploy.kubevirt.master
-    master_username = deploy.kubevirt.master_user
-    master_password = deploy.kubevirt.master_passwd
+    master = deploy.kubevirt.endpoint
+    master_username = ''
```
Comment (hkx303) [inline utils/installer.py:262]: why do we still need the `master_username` param?

File: utils/installer.py [hunk 2]
```diff
@@ -258,17 +258,17 @@ def setup_libvirt():
...
+    master_password = deploy.kubevirt.token
     guest_name = deploy.kubevirt.guest_name
     guest_username = deploy.kubevirt.guest_user
     guest_password = deploy.kubevirt.guest_passwd
     ssh_master ={
```
Comment (hkx303) [inline utils/installer.py]: is it necessary to remain this dict?

**Review by hkx303 — APPROVED** [review]
ACK, pending some small questions

File: utils/installer.py (hunk 1 above)
Comment (eko999) [reply utils/installer.py:262]: It's not necessary, but let's remain them for the installer.py as an uniform format.

File: utils/installer.py (hunk 2 above)
Comment (eko999) [reply utils/installer.py]: ssh_master is not needed, will remove it.

**Comment by Junefen** [comment]
ACK

**Pull request merged**